app/helpers/portal/service_requests_helper.rb

```diff
@@ -57,8 +57,8 @@ def populate_fulfillment_time fulfillment
     end
   end
 
-  def default_display workflow_state
-    workflow_state == 'submitted' ? '' : 'display:none;'
+  def default_display workflow_state, desired_status
```

Comment:
Jason,

Can you jump on chat?

Thanks!

Andrew – to be clear – you don’t think this feature was done properly in the code. Correct?

From: Andrew Cates [mailto:notifications@github.com]
Sent: Friday, July 17, 2015 7:55 AM
To: sparc-request/sparc-request
Subject: Re: [sparc-request] Admin remember status (#112)




In app/helpers/portal/service_requests_helper.rb<https://github.com/sparc-request/sparc-request/pull/112#discussion_r34882803>:

> @@ -57,8 +57,8 @@ def populate_fulfillment_time fulfillment

>      end

>    end

>

> -  def default_display workflow_state

> -    workflow_state == 'submitted' ? '' : 'display:none;'

> +  def default_display workflow_state, desired_status

Why is workflow_state even being passed in if all we are using is desired_status? I don't think this is done properly.

—
Reply to this email directly or view it on GitHub<https://github.com/sparc-request/sparc-request/pull/112/files#r34882803>.